src/compiler/diagnosticMessages.json

```diff
@@ -1388,6 +1388,10 @@
         "category": "Message",
         "code": 1449
     },
+    "Dynamic import must only have a specifier and an optional assertion as arguments": {

```

Comment:
```suggestion
    "Dynamic imports can only accept a path and an optional assertion as arguments": {
```

Reply:
Agree, but I think `module specifier` is a more correct term than `path`